Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "linjector-rs"
-version = "0.1.1-dev"
+version = "0.1.2-dev"
```

```suggestion
version = "0.1.1-dev"
```

Version will be bumped on main.